presto-testing/src/main/java/io/prestosql/testing/AbstractTestJoinQueries.java

```diff
@@ -2256,6 +2256,25 @@ public void testRightJoinWithEmptyProbeSide()
                 "WITH small_part AS (SELECT * FROM part WHERE name = 'a') SELECT lineitem.orderkey FROM small_part RIGHT JOIN lineitem ON  small_part.partkey = lineitem.partkey");
     }
 
+    @Test
+    public void testEquijoinOnDifferentTypesWithFilter()
+    {
+        // Non-sensical query that exercises a join on integer vs bigint with a filter on the integer column that can be pushed down
```

Comment:
remove "Non-sensical". Most test queries do not make much sense

Reply:
It’s particularly non-sensical because it joins nation with lineitem on keys that are unrelated. But fair enough, I’ll remove it.